hygieia-jenkins-plugin/src/main/java/jenkins/plugins/hygieia/HygieiaGlobalListener.java

```diff
@@ -114,10 +118,24 @@ public void onCompleted(Run run, @Nonnull TaskListener listener) {
 
         boolean showConsoleOutput = hygieiaGlobalListenerDescriptor.isShowConsoleOutput();
         BuildStatus buildStatus = HygieiaUtils.getBuildStatus(run.getResult());
+        String jobUrl = HygieiaUtils.getJobUrl(run);
+        String wfapiUrl = jobUrl + WFAPI_RUNS;
+        listener.getLogger().println("Hygieia: wfapi url : " + wfapiUrl);
+        LinkedList<BuildStage> buildStages=null;
+        String responseString = "";
+        try{
+            RestCall.RestCallResponse callResponse = hygieiaService.getStageResponse(wfapiUrl,hygieiaGlobalListenerDescriptor.getJenkinsUserId(),hygieiaGlobalListenerDescriptor.getJenkinsToken());
+            listener.getLogger().println("Hygieia: call response code : " + callResponse.getResponseCode());
+            if(Objects.nonNull(callResponse)){
+                responseString = callResponse.getResponseString();
+                buildStages=  HygieiaUtils.getBuildStages(responseString);
+            }
+        }catch (Exception e){
+            listener.getLogger().println("Hygieia: call response error : " + e.getStackTrace());
+        }
```

Comment:
Should the catch block return with exception? How does it flow after catch exception and then goes to the next line?

Reply:
@tabladrum  in this case, catch does nothing but to log error to console and go to next line. Catch should probably throw exception to calling method to cascade to top.

Reply:
@tabladrum  I am thinking if we cascade exception to the calling method till top, will that cause pipeline build to abort and fail?